.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
can you commit these config files locally?

Reply:
Okay, got it